pkgs/development/python-modules/mastodonpy/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, fetchPypi
+, decorator
+, six
+, requests
+, pytz
+, dateutil
+, python_magic
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "Mastodon.py";
+  version = "1.4.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1229wbckanp08bbqajsn31ggvay271d883nca28wpvsaqdsv8ci5";
+  };
+
+  propagatedBuildInputs = [ decorator six requests pytz dateutil python_magic ];
+
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
+    py.test
+  '';
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.